nsv13/code/modules/munitions/ammunition/countermeasure_ammo.dm

```diff
@@ -1,10 +1,60 @@
 /obj/item/ship_weapon/ammunition/countermeasure_charge //A single use of the countermeasure system
-	name = "\improper Countermeasure Tri-Charge" //temp
-	desc = "A tri-charge of countermeasure chaff for a fighter" //temp
+	name = "\improper Countermeasure Tri-Charge"
+	desc = "A tri-charge of countermeasure chaff for a fighter"
 	icon = 'nsv13/icons/obj/munitions.dmi'
 	icon_state = "iff" //temp
-	w_class = 4
+	var/restock_amount = 3
+	w_class = 3
 
-/obj/item/ship_weapon/ammunition/countermeasure_charge/Initialize()
-	..()
-	AddComponent(/datum/component/twohanded/required)
+/obj/item/ship_weapon/ammunition/countermeasure_charge/five
+	name = "\improper Countermeasure Penta-Charge"
```

Comment:
This still fires in bursts of three, so it's still a tri-charge, right? Maybe a "high-capacity countermeasure tri-charge" or something